docs/help.md

```diff
@@ -0,0 +1,143 @@
+# So you want to help out
+
+First of all: thank you! Shipping complicated native code into Atom is risky, and every bug we find now saves scores of Atom users from unceremonious crashes or lock-ups. I'm going to be working on some more intensive randomized testing soon, but having multiple pairs of eyes on this is critical to avoid the pitfalls of "works on my machine" syndrome.
+
+I've written a quick Swiss-army-knife utility to exercise @atom/watcher and compare it to other watching libraries at [@smashwilson/watcher-stress](https://github.com/smashwilson/watcher-stress). You can clone it and run it from that repository:
+
+```bash
+$ git clone git@github.com:smashwilson/watcher-stress.git
+$ cd watcher-stress
+$ npm_config_debug="true" npm install
+
+# ...
+
+$ script/watcher-stress --help
+```
+
+Or, you can install it directly from npm:
+
+```bash
+$ npm_config_debug="true" npm install -g @smashwilson/watcher-stress
+
+# ...
+
+$ watcher-stress --help
+```
+
+Or even run it directly from `npx`, although that will recompile all of its native dependencies on each invocation:
+
+```bash
+$ npm_config_debug="true" npx @smashwilson/watcher-stress --help
+
+# ...
+```
+
+## Ways to help
+
+Before you begin:
+
+```bash
+# On MacOS and Linux, be sure you're generating core files in this shell session.
+$ ulimit -c unlimited
+
+# Create a directory to write logs to *that you aren't watching.*
+$ mkdir -p ~/watcher-logs
```

Comment:
Since I doubt the user's `$HOME` directory will be missing, the `-p` isn't necessary, and removing it allows this command to work on Windows (in PowerShell at least).